scipy/special/ellint_carlson_wrap.cxx

```diff
@@ -0,0 +1,158 @@
+#include "ellint_carlson_wrap.hh"
+
+
+#undef _BEGIN_EXTERN_C
+#undef _END_EXTERN_C
+#ifdef __cplusplus
+#define _BEGIN_EXTERN_C	extern "C" {
+#define _END_EXTERN_C	}
+#else
+#define _BEGIN_EXTERN_C	/* nothing */
+#define _END_EXTERN_C	/* nothing */
+#endif
+
+
+#include "sf_error.h"
+
+
+static constexpr double ellip_rerr = 5e-16;
+
+
+_BEGIN_EXTERN_C
+
+
+double fellint_RC(double x, double y)
```

Comment:
Simple changes to this file to make it [PEP7](https://www.python.org/dev/peps/pep-0007/) compliant -- I believe we are trying to use this coding standard for C++ as well, it's just not been well documented or brought up during reviews